packages-exp/auth-compat-exp/demo/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I'm guessing you didn't mean to commit this

Reply:
done, good catch